scavenger-api/src/main/kotlin/com/navercorp/scavenger/dto/ExportMethodInvokationDto.kt

```diff
@@ -0,0 +1,29 @@
+package com.navercorp.scavenger.dto
+
+import com.navercorp.scavenger.entity.ExportMethodInvocationEntity
+
+data class ExportMethodInvokationDto(
+    val id: Long,
+    val modifiers: String,
+    val signature: String,
+    val createdAt: String,
+    val lastSeenAtMillis: String,
+    val invokedAtMillis: String,
+    val status: String,
+    val timestamp: String
+) {
+    companion object {
+        fun from(entity: ExportMethodInvocationEntity): ExportMethodInvokationDto {
+            return ExportMethodInvokationDto(
+                entity.id,
+                entity.modifiers,
+                entity.signature,
+                entity.createdAt.toString(),
+                entity.lastSeenAtMillis.toString(),
+                entity.invokedAtMillis.toString(),
+                entity.status,
+                entity.timestamp.toString()
```

Comment:
And please use named parameter here.

Reply:
I fixed it on commit 1aed024. Thank you for your reviews!